pages/models.py

```diff
@@ -389,13 +389,22 @@ class LiveEvent(models.Model):
         help_text=_("Date/time the live event was created (in ISO format)"))
     scheduled_at = models.DateTimeField(
         help_text=_("Date/time the live event is scheduled (in ISO format)"))
+    rank = models.PositiveSmallIntegerField(default=1,
+        help_text="Unique integer to denote the index of the LiveEvent")
     location = models.URLField(_("URL to the calendar event"), max_length=2083)
     max_attendees = models.IntegerField(default=0)
+    status = models.CharField(
```

Comment:
Put the choices as constant defined at the class level. See definition of [FORMAT_CHOICES](https://github.com/djaodjin/djaodjin-pages/blob/096eee41cbbe4e5fd4d2cc0a2c9ab68ce76e57d4/pages/models.py#L172) as example.

Reply:
I meant write code like:

```
   CANCELED = 'Canceled'
   SCHEDULED = 'Scheduled'

   EVENT_STATUS_CHOICES = (
     (SCHEDULED, 'Scheduled'),
     (CANCELED, 'Canceled')
   )
```

such that you write `live_event.status = LiveEvent.CANCELED`

The idea is that using a named "constant", it is a lot easier to maintain the code than if strings are used all over the place. Ideally you would define `SCHEDULED = 0` and `status = models.SmallPositiveInteger` to speed up SQL queries in the database.